packages/react/src/primitives/types/selectField.ts

```diff
@@ -5,4 +5,9 @@ import { FieldProps } from './field';
 export interface SelectFieldProps
   extends FieldProps,
     FlexContainerStyleProps,
-    SelectProps {}
+    SelectProps {
+  /**
+   * Accepts an array of strings which maps to the value, label and children of each option tag in the SelectField
```

Comment:
It's probably unnecessary to mention the types in the description, since they are already defined in the type. How about something like: 
```suggestion
   * List of option values for select dropdown
```